js/components/LetterImage.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+
+/* LetterImage - displays a letter image retrieved from
+ * a URL that points to the image on the backend server.
+ * It also resizes the image according to the size specified
+ * in the configuration form, maintaining the aspect ratio
+ * of the original.
+ */
+
+const SMALL_DIM = 25;
```

Comment:
I think this is fine for now, but we probably want to look at using em/rem or at least vh/vw at some point.